### PR TITLE
Homepage new copy

### PR DESCRIPTION
**Pull request by jameshupp**
## Dependencies
 - [x] Merge #661 into the `homepage_new_copy` branch.
 - [x] Merge [cg-style 237](https://github.com/18F/cg-style/pull/237) into `master` in that repo.

## Changes
* New copy for the homepage, streamlined and focused on customer business. We’re reserving all details about how the platform works and the technology it’s based on to internal content.
* Updated contact.html, which is a partial that at some point we probably need to rename. It includes a number of things, but it’s basically the full set of “next steps” a person might take who is interested to some degree or another
* Lower case ‘g’ in the government banner at the top. This is consistent across 18F products. That’s not necessarily a reason to do it in cloud.gov, but if nobody has a reason to oppose it I want to keep this change.

## Review
* @nikzei 
* @vz3 (or anyone else from Navigator — see Notes below)

## Notes
* I’m going to file an issue for the folks from Navigator momentarily. There are a couple design issues that the new content presents for the homepage. *Please do not merge until we incorporate those changes.*

### CODE REVIEW

**Comment by jameshupp** [comment]
@thisisdano @msecret @nikzei 

I added two checkboxes at the beginning of this PR's first comment. This will be ready to merge when those are handled, so if you can check them off when ready that'd be great. I'll follow up on anything outstanding next week.

File: layouts/index.html
Comment (adborden) [inline layouts/index.html]: typo "can can"

File: layouts/index.html
Comment (adborden) [inline layouts/index.html]: These sound like two completely separate ideas. Shouldn't they be their own line items?

File: layouts/index.html
Comment (adborden) [inline layouts/index.html]: How about "any foothold a bad actor _might_ have found"?

File: layouts/partials/header.html
Comment (adborden) [inline layouts/partials/header.html]: Shouldn't Federal Government be lowercase, too, since it's not a title/name?

File: layouts/index.html
Comment (adborden) [inline layouts/index.html]: @thisisdano would you resolve the merge conflicts and fix this typo ^^

File: layouts/index.html
Comment (thisisdano) [reply layouts/index.html]: Ha, that sounds reasonable.

File: layouts/index.html
Comment (thisisdano) [reply layouts/index.html]: Seems reasonable.

File: layouts/partials/header.html
Comment (thisisdano) [reply layouts/partials/header.html]: in the alt text? seems like a good call.

**Comment by thisisdano** [comment]
@adborden I think this is ready to go now. Unfortunately, I added a utility class to the homepage billboard H1, as something must've gotten screwed up in the regular css classes, but I wanted it to look right right now, and not wait for a new cg-style release. I can mark this to fix later.

**Review by adborden — APPROVED** [review]

**Pull request merged**
